### ISSUE TITLE
Error on Get-UnifiSiteDevice

### ISSUE THREAD

**Issue by ccampbelljr** [MichaelMcCool/Unifi#1]
If I run:

`Get-UnifiSiteDevice -name a6264rzr -Detailed | select Name,Type,Ip`

I get this error on some sites: 
![image](https://user-images.githubusercontent.com/83838295/117495856-2cd92f80-af44-11eb-954c-3c8087112f32.png)

I've confirmed the device output is complete (no missing devices). For reference, that screenshot above was produced on a site with 50 devices, 2 of which are offline (and 1 of those is disabled). But there are 8 errors. I've looked at the two other sites producing the same errors and there are no offline/disabled devices, so that doesn't appear to be the common denominator.

If I comment out lines 53-56 in _Get-UnifiSiteDevices_ and run the same thing, it outputs as expected.
![image](https://user-images.githubusercontent.com/83838295/117496363-eb954f80-af44-11eb-8406-ed9aaedb9c98.png)

**Comment by MichaelMcCool** [comment]
When I was testing this command, I found the basic results were not an actual subset of detailed, so when you run detailed, it runs a basic query in the background and then adds in the missing 'disabled' value. It appears that the missing 'disabled' value from the detailed response may have been fixed by Unifi in a recent firmware release.

I'll add in a check to only add it if it is missing. The errors displayed won't actually affect the final output, it will just look horrible.
